doc/dev/osd_internals/erasure_coding/ecbackend.rst

```diff
@@ -185,15 +185,15 @@ from the pipeline.
 
 See ExtentCache.h for a detailed explanation of how the cache
 states correspond to the higher level invariants about the conditions
-under which cuncurrent operations can refer to the same object.
+under which concurrent operations can refer to the same object.
 
 Pipeline
 --------
 
 Reading src/osd/ExtentCache.h should have given a good idea of how
 operations might overlap.  There are several states involved in
 processing a write operation and an important invariant which
-isn't enforced by PrimaryLogPG at a higher level which need to be
+isn't enforced by PrimaryLogPG at a higher level which needs to be
 managed by ECBackend.  The important invariant is that we can't
 have uncacheable and rmw operations running at the same time
```

Comment:
can you also fix `s/rmw/read–modify–write`  rmw might not be clear to beginner

Reply:
Sure!